packages/table/src/store/index.js

```diff
@@ -70,6 +70,9 @@ Watcher.prototype.mutations = {
   sort(states, options) {
     const { prop, order } = options;
     if (prop) {
+      if (states.sortingColumn) {
```

Comment:
You can add these lines in `$nextTick` callback.

Reply:
I moved fix in another function.